src/frontend/components/InstructorWrapper/InstructorWrapper.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import '../../testSetup';
+
+import { shallow } from 'enzyme';
+import * as React from 'react';
+
+// Mock the state context before we import InstructorWrapper. We'll be able to mutate the
+// context object to vary it for every test
+const context: {
+  state: string;
+} = { state: '' };
+jest.doMock('../App/App', () => {
+  return {
+    AppDataContext: {
+      Consumer: (props: any) => props.children(context),
+    },
+  };
+});
+
+import { appState } from '../../types/AppData';
+import { InstructorView } from '../InstructorView/InstructorView';
+import { InstructorWrapper } from './InstructorWrapper';
+
+describe('<InstructorWrapper />', () => {
+  it('wraps its children in an instructor view if the current user is an instructor', () => {
+    context.state = appState.INSTRUCTOR;
+    const wrapper = shallow(
+      <InstructorWrapper>
+        <div className="some-child" />
+      </InstructorWrapper>,
+    );
+
+    expect(
+      wrapper
+        .dive()
+        .find(InstructorView)
+        .exists(),
+    ).toBeTruthy();
```

Comment:
`exists` return a boolean, why not asserting the value is `true` ? It is less ambiguous than `truthy` IMO